connection_test.go

```diff
@@ -314,3 +315,20 @@ func fetchResultByQueryID(t *testing.T, customget FuncGetType, expectedFetchErr
 	}
 	return nil
 }
+
+func TestPrivateLink(t *testing.T) {
+	_, err := buildSnowflakeConn(context.Background(), Config{
+		Account:  "testaccount",
+		User:     "testuser",
+		Password: "testpassword",
+		Host:     "testaccount.us-east-1.privatelink.snowflakecomputing.com",
+	})
+	if err != nil {
+		t.Error(err)
+	}
+	ocspURL := os.Getenv(cacheServerURLEnv)
+	expectedURL := "http://ocsp.testaccount.us-east-1.privatelink.snowflakecomputing.com/ocsp_response_cache.json"
+	if ocspURL != expectedURL {
+		t.Errorf("expected: %v, got: %v", expectedURL, ocspURL)
```

Comment:
For completion check the retry url as well. Also is there a way you can test the retry url generated with the fields populated?